js/bitopro.js

```diff
@@ -0,0 +1,1151 @@
+        //             "remainingAmount":"0.01",
+        //             "executedAmount":"0",
+        //             "fee":"0",
+        //             "feeSymbol":"twd",
+        //             "bitoFee":"0",
+        //             "total":"0",
+        //             "seq":"BNBTWD548774666",
+        //             "timeInForce":"GTC",
+        //             "createdTimestamp":1644898944074,
+        //             "updatedTimestamp":1644899002598
+        //         }
+        //
+        let id = this.safeString (order, 'orderId');
+        if (id === undefined) {
+            id = this.safeString (order, 'id');
+        }
+        const timestamp = this.safeInteger (order, 'timestamp');
+        const side = this.safeString (order, 'action');
+        const amount = this.safeString (order, 'amount');
+        const price = this.safeString (order, 'price');
+        const marketId = this.safeString (order, 'pair');
+        if (marketId !== undefined) {
+            market = this.market (marketId);
+        }
+        const symbol = this.safeString (market, 'symbol');
+        const orderStatus = this.safeString (order, 'status');
+        const status = this.parseOrderStatus (orderStatus);
+        const type = this.safeString (order, 'type');
+        const average = this.safeString (order, 'avgExecutionPrice');
+        const filled = this.safeString (order, 'executedAmount');
+        const remaining = this.safeString (order, 'remainingAmount');
+        const timeInForce = this.safeString (order, 'timeInForce');
+        let fee = undefined;
+        const feeAmount = this.safeString (order, 'fee');
+        const feeSymbol = this.safeString (order, 'feeSymbol');
+        if (feeAmount !== undefined) {
+            fee = {
+                'currency': feeSymbol,
+                'cost': feeAmount,
+            };
+        }
+        return this.safeOrder ({
+            'id': id,
+            'clientOrderId': undefined,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'symbol': symbol,
+            'type': type,
+            'timeInForce': timeInForce,
+            'postOnly': undefined,
+            'side': side,
+            'price': price,
+            'stopPrice': undefined,
+            'amount': amount,
+            'cost': undefined,
+            'average': average,
+            'filled': filled,
+            'remaining': remaining,
+            'status': status,
+            'fee': fee,
+            'trades': undefined,
+            'info': order,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'type': type,
+            'pair': market['id'],
+            'action': side,
+            'amount': this.numberToString (amount),
```

Comment:
Probably not our fault but there is something wrong with the limits, the minimum amount for the "USDC/TWD" pair is 0.01 but I can't create market orders using values bigger than that (0.1, 0.2,...), limit orders works though

![image](https://user-images.githubusercontent.com/43336371/154239513-4f5de722-33e2-4d7d-9833-481cb21687b6.png)



![image](https://user-images.githubusercontent.com/43336371/154239209-f3ee8696-de8b-418e-9fb2-e5b66f77c469.png)

Reply:
I had this error too. They might have another limit for the opposite side (TWD).

Reply:
I had this error too. They might have another limit for the opposite side (TWD).

Reply:
@sc0Vu Nonetheless, this should be taken care of because users are not expecting this behavior